webapp/channels/src/components/menu/menu.tsx

```diff
@@ -65,14 +65,17 @@ type MenuProps = {
 const defaultAnchorOrigin = {vertical: 'bottom', horizontal: 'left'};
 const defaultTransformOrigin = {vertical: 'top', horizontal: 'left'};
 
-type VerticalOrigin = 'top' | 'center' | 'bottom';
-type HorizontalOrigin = 'left' | 'center' | 'right';
+type VerticalOrigin = 'top' | 'center' | 'bottom' | number;
+type HorizontalOrigin = 'left' | 'center' | 'right' | number;
```

Comment:
Are these still needed? I don't see you passing any numbers for these elsewhere

Reply:
No, they aren't needed. Reverted.